src/NzbDrone.Core/DecisionEngine/Specifications/SeasonPackOnlySpecification.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using NLog;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.IndexerSearch.Definitions;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Common.Extensions;
+using System.Linq;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.DecisionEngine.Specifications
+{
+    public class SeasonPackOnlySpecification : IDecisionEngineSpecification
+    {
+        private readonly IConfigService _configService;
+        private readonly Logger _logger;
+
+        public SeasonPackOnlySpecification(IConfigService configService, Logger logger)
+        {
+            _configService = configService;
+            _logger = logger;
+        }
+
+        public SpecificationPriority Priority => SpecificationPriority.Default;
+        public RejectionType Type => RejectionType.Permanent;
+
+        public virtual Decision IsSatisfiedBy(RemoteEpisode subject, SearchCriteriaBase searchCriteria)
+        {
+            if (searchCriteria == null)
+            {
+                return Decision.Accept();
+            }
+
+            if (searchCriteria.Episodes.Count == 1) return Decision.Accept();
```

Comment:
Style wise I'd make this like `searchCriteria == null` and wrap the return in braces